meta-mender-rockchip/recipes-bsp/grub-mender-grubenv/files/02_tinkerboard_console_grub.cfg

```diff
@@ -0,0 +1 @@
+set console_bootargs="console=ttyS2,115200n8"
```

Comment:
What is the reasoning behind this change?

I assume the default will output console on "serial + hdmi" during U-boot and in user-space, and kernel boot log is only printed on HDMI.

I am not sure if we should change this.

Reply:
In my case I had neither serial or HDMI console without this. I'll test again with a clean build but it looks like something is not right by default